tally_ho/apps/tally/templates/super_admin/form_results_duplicates.html

```diff
@@ -41,17 +41,17 @@ <h1>{% trans 'Results Duplicated List' %}</h1>
 <div class="pagination">
     <span class="step-links">
         {% if forms.has_previous %}
-            <a href="?page={{ forms.previous_page_number }}">previous</a>
+            <a href="?page={{ forms.previous_page_number }}">{% trans 'previous' %}</a>
         {% endif %}
 
         {% if forms %}
         <span class="current">
-            Page {{ forms.number }} of {{ forms.paginator.num_pages }}.
+            {% trans 'Page'%}  {{forms.number}} {%  trans 'of' %} {{forms.paginator.num_pages}}.
```

Comment:
Not sure if/how it can be done, but curious if its possible to use string interpolation and then translate the full string as a single phrase with placeholders, I think that way you can also account for things like rtl text in a more consistent way

Reply:
Cool makes sense, a [blocktranslate template tag](https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#blocktranslate-template-tag) should do the trick.

Reply:
Done